frontends/web/src/routes/lightning/receive/receive.tsx

```diff
@@ -249,7 +253,9 @@ export function Receive() {
           <Status type="warning" hidden={!receiveError}>
             {receiveError}
           </Status>
-          <Header title={<h2>{t('lightning.receive.title')}</h2>} />
+          <Header title={<h2>{t('lightning.receive.title')}</h2>}>
+            <HideAmountsButton />
+          </Header>
```

Comment:
I don't think it needs to be hidden on lightning receive, as the user will probably show this invoice to somebody else, who will see and pay the amount anyway. but maybe there is a use case that I am missing 🤔